src/renderer/App.vue

```diff
@@ -155,7 +155,7 @@ export default {
       return this.$store.getters['session/background'] || `wallpapers/${this.hasSeenIntroduction ? 1 : 2}Default.png`
     },
     hasAnyProfile () {
-      return !!this.$store.getters['profile/all'].length
```

Comment:
I feel like the previous line was a better check, so maybe look into the `profile/all` store module to see what can be changed there instead. Could you then also add tests for that scenario please (if the file is missing, just add tests for the method you change)

Reply:
Well... anyway the code  `return this.$store.getters['session/profile']` will return an active profile. 
@alexbarnsley maybe change the computed property from `hasAnyProfile` to `hasProfile`?

Reply:
Yeh can do. It's not a major change. In theory `session/profile` should always be set so it should be okay. Could you add some tests for this please. There isn't currently a test file so just add the mounting stuff and test this particular computed method. Thanks!

Reply:
@brenopolanski 👀